### ISSUE TITLE
As a SO.CE I would like to create a properly formatted table for accessibility purposes

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1551]
## Context

Our current WYSIWYG configuration strips the table <caption> tag, the table <tbody> tag,  as well as the summary attribute.

## Process

[ordered list the process to finding and recreating the issue, example below]

1. Login to site as a [SO, CE, EO, ADMIN or DEV]
2. Click Add Content
3. Click Basic Page
4. Paste the table code below
5. Click save

`<table  summary="table summary here"><caption>table caption here</caption><tbody><tr><td>Table Cell 1</td><td>Table Cell 2</td></tr></tbody></table>`

## Expected result

The table should render properly with the caption above it and the table summary attribute should be present.

## Current result

The table caption and summary are removed are removed.

## Possible Fix

Update the wysiwyg filter configuration

**Comment by cathysnider** [comment]
Looks good in 2.8.0 
https://www-test.colorado.edu/sandbox/cathy/design-examples/table-styles
